src/ucs/debug/log.c

```diff
@@ -320,26 +425,56 @@ void ucs_log_init()
 
     ucs_log_initialized = 1; /* Set this to 1 immediately to avoid infinite recursion */
 
+    if (ucs_global_opts.log_file_max_size < ucs_log_get_buffer_size()) {
+        ucs_fatal("the maximal log file size (%zu) has to be >= %zu",
```

Comment:
please describe this in the definition of `LOG_FILE_MAX_SIZE`

Reply:
done